titanic.ipynb

```diff
@@ -5,14 +5,16 @@
       "cell_type": "markdown",
```

Comment:
_Commented on notebook `titanic.ipynb` Cell 2 From line 15 to 16_
```python
plt.xlabel('Person Type')
plt.ylabel('Count')
```
looks good

_Respond and view the context [here](https://gitnotebooks.com/smith-kyle/notebook-diff-tests/pull/32)._

Reply:
response in githb